boards/common/weact-f4x1cx/include/board.h

```diff
@@ -27,24 +27,16 @@ extern "C" {
 
 #include "mtd.h"
 #include "periph_cpu.h"
+#include "timex.h"
```

Comment:
This include doesn't seem to be needed here.

Reply:
moved it to board.c, needed for `US_PER_MS` in `mtd_spi_nor_params_t`